uber/custom_tags.py

```diff
@@ -49,11 +57,15 @@ def event_dates():
 
 @JinjaEnv.jinja_export
 def hour_day_local(dt):
+    if not dt:
+        return ''
     return hour_day_format(dt)
```

Comment:
Internally `hour_day_format` is already converting to the event timezone, so doing it again would be redundant :(

I could have exported `hour_day_format` directly as a Jinja function, but the one thing this function does that `hour_day_format` doesn't do is check for an empty `dt` argument.